docs/08-Customization.md

```diff
@@ -179,6 +179,25 @@ Passed configuration is accessible via `config` property: `$this->config['param'
 
 Check out a very basic extension [Notifier](https://github.com/Codeception/Notifier).
 
+### Custom Commands
+
+You can add your own commands to codeception.
+
+Your custom commands have to implement the interface Codeception\CustomCommandInterface,
+because there has to be a function to get the name of the command.
+
+You have to register your command in the file `codeception.yml` 
+
+```yaml
+extensions:
+    commands: [Codeception\Command\MyCustomCommand]
+```
+
+If you want to activate the Command globally, because you use more then one ```codeception.yml``` file, then
+you have to register your command in ```codeception.dist.yml``` in the root folder of your project.
+
+You can find an example here: [/tests/data/register_command/examples/MyCustomCommand.php]
```

Comment:
such permalink won't work on our site. However, before it's merged to master we can't get such link working (
anyway, let's keep it as it is and update once we need it. Maybe we can have a gist for now?